src/Sulu/Bundle/SnippetBundle/Tests/Functional/Controller/SnippetControllerTest.php

```diff
@@ -76,6 +77,26 @@ public function testGetMany()
         $res = reset($res['_embedded']['snippets']);
         $this->assertEquals('Le grande budapest', $res['title']);
         $this->assertEquals($this->hotel1->getUuid(), $res['id']);
+        $this->assertEquals('Hotel', $res['localizedTemplate']);
+    }
+
+    public function testGetManyLocalizedTemplate()
+    {
+        $this->client->request(
+            'GET',
+            sprintf(
+                '/snippets?ids=%s&language=fr',
+                $this->hotel1->getUuid()
+            )
+        );
+        $response = $this->client->getResponse();
+
+        $this->assertEquals(200, $response->getStatusCode());
+        $res = json_decode($response->getContent(), true);
+
+        $res = reset($res['_embedded']['snippets']);
```

Comment:
Do not abbreviate. Is that also done in the other test cases? Can you please change that everywhere?